docs/content/docs/start/tutorial.md

```diff
@@ -7,188 +7,156 @@ aliases:
-    const wrapperDiv = document.createElement("div");
-    wrapperDiv.style.textAlign = "center";
-    div.append(wrapperDiv);
-    const diceCharDiv = document.createElement("div");
-    diceCharDiv.style.fontSize = "200px";
-    const rollButton = document.createElement("button");
-    rollButton.style.fontSize = "50px";
-    rollButton.textContent = "Roll";
-
-    rollButton.addEventListener("click", () => { console.log("Roll!"); });
-    wrapperDiv.append(diceCharDiv, rollButton);
-
-    const updateDiceChar = () => {
-        const diceValue = 1;
-        // Unicode 0x2680-0x2685 are the sides of a die (⚀⚁⚂⚃⚄⚅).
-        diceCharDiv.textContent = String.fromCodePoint(0x267F + diceValue);
-        diceCharDiv.style.color = `hsl(${diceValue * 60}, 70%, 50%)`;
-    };
-    updateDiceChar();
-}
-```
+Lastly, `root` defines the HTML element that the Dice will render on.
 
-## Define the model interface
+```js
+const client = new AzureClient(localConfig);
 
-To clarify what your model needs to support, start by defining its public interface.
+const containerConfig = {
+      initialObjects: { diceMap: SharedMap }
+  };
 
-```ts
-export interface IDiceRoller extends EventEmitter {
-    readonly value: number;
-    roll: () => void;
-    on(event: "diceRolled", listener: () => void): this;
-}
+const root = document.getElementById("root")
 ```
 
-As you might expect, you have the ability to read its value and command it to roll. However, you also need to declare an event `"diceRolled"` in your interface. Fire this event whenever the die is rolled (using [EventEmitter](https://nodejs.org/api/events.html#events_class_eventemitter)).
+## Create a Fluid container
 
-This event is particularly important because you're building a collaborative experience. It's how each client will observe that other clients have rolled the die remotely, so they know to update with the new value.
+Fluid data is stored within containers, and these containers need to be created before other users can load them. Since creation and loading of containers both happen in the browser, a Fluid application needs to be capable of handling both paths.
 
-## Implement the model
+### Create a new container
 
-Up to this point, you've used TypeScript. To implement the model for your collaborative DiceRoller, you'll now use features from the Fluid Framework.
+The create path of the application starts with calling `createContainer` and passing in a schema defining which shared objects will be available on the new `container`.
 
-The Fluid Framework provides a class called **[DataObject][]** which you can extend to build your model. You'll use a few features from DataObject, but first take a look at the following code:
+After setting some default values on the `diceMap`, the container is attached by calling the `attach` function. Attaching allows changes to be sent to and from the client, and returns the assigned container id that will be used in the load path.
```

Comment:
"client" so far has been used for tinylicious client, and "service" hasn't really been used at all. Other "collaborators"? attached to a "client"?

Reply:
"Service" is most correct here.  "Attach" is kind of a weird verb for what it means but it's what we have right now.

Reply:
should we instead be importing `tinylicious-service`? Calling it a service but interfacing it through a `const client` make it pretty confusing.

Reply:
The service is remote, running on a server in the cloud, so it wouldn't make sense to import the service itself.  Instead you are importing a (local) client object which knows how to communicate with that service.

Reply:
Fluid requires a service. To connect to a specific service you need to use the corresponding *-client package. Attaching is the act of creating/registering a new container with the corresponding service instance. 

@tylerbutler, your comment feels a bit backwards because until you attach there's not way to get the id to have another editor. Maybe something like...

> Changes to the dice value will be local until the container is *attached*. Once attached, the app begins sending and receiving operations to and from the service. The `attach` call returns the `id` of the container which other clients can use to load and simultaneously edit against the container.